### PR TITLE
Move status endpoint from /status to / for all services

### LINKED ISSUE #123: Refactor root endpoint
Right now, all services have a /status endpoint whose main purpose is health checks. It'd be nice to refactor this to the root directory "/" so it coincides with the default entry point for every service

### PR DESCRIPTION
**Pull request by hkeeler**
Resolves #123

This PR is dependent on https://github.com/cfpb/grasshopper-parser/pull/24.  Please review and merge that PR first.

### CODE REVIEW

File: project/GrasshopperBuild.scala [hunk 1]
```diff
@@ -117,7 +117,6 @@ object GrasshopperBuild extends Build {
     .configs( IntegrationTest )
     .settings(buildSettings: _*)
     .settings(
-      Revolver.settings ++
```
Comment (wpears) [inline project/GrasshopperBuild.scala:120]: Just curious--why did you need to remove this line?

Comment (hkeeler) [inline project/GrasshopperBuild.scala:120]: Thanks, I meant to add a comment about this.  `Revolver` adds the `~re-start` feature to `sbt`, which listens for code changes, and re-deploys the apps.   The `client` module is just a lib, and has no "main" method, so `Revolver` doesn't know what to do with it, and breaks `re-start` when running it for the whole `grasshopper` project.

**Comment by wpears** [comment]
:+1:
